showcase/src/components/sidebar-link.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+
+import { type NavigationItem } from "@/lib/navigation";
+import { cn } from "@/lib/utils";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import { memo } from "react";
+
+export const SidebarLink = memo(function SidebarLink({
```

Comment:
Hey while you're here, can you removethe `memo` here? This component is simple enough that it should be faster NOT to memoize it